### DISCUSSION TITLE
Why suneditor always add extra inline css ?

### DISCUSSION THREAD

**Discussion by iamluxan** [JiHong88/SunEditor#1170] [Q&A]
Hi,
Why SunEditor always add extra CSS to my textarea text ?

Example :
<span style="font-weight: var(--bs-body-font-weight); text-align: var(--bs-body-text-align);">

<p><span style="font-weight: var(--bs-body-font-weight); text-align: var(--bs-body-text-align);">Le chardonneret élégant est également un </span><strong><span style="text-align: var(--bs-body-text-align);">oiseau très vocal</span></strong><span style="font-weight: var(--bs-body-font-weight); text-align: var(--bs-body-text-align);">, même l&apos;hiver, avec un chant mélodieux et agréable à écouter. On distingue facilement son cri typique fait de « didelit ». Le cri d&apos;alarme est l&apos;onomatopée « wèèli » et lorsqu&apos;il s&apos;en prend à un de ses rivaux, il émet des « tschrr » sonores.</span></p>

**Comment by JiHong88** [comment]
Sorry, I don't understand.. what mean?

**Reply by monkeydev001** [reply]
I think he means the CSS styles, like:
`style="color: rgb(0, 76, 255); text-decoration: none; outline: none !important; cursor: pointer !important; box-sizing: border-box; -webkit-user-drag: none; overflow: visible; font-family: inherit; font-size: inherit;"`

How do I disable adding attributes to HTML tags?
